doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -75,7 +78,7 @@ KRKNWK-19647: nRF53 has high power consumption in sleep mode
 
   **Affected platforms:** nRF5340
 
-.. rst-class:: v2-9-0-nRF54H20-1 v2-9-1 v2-9-0 v2-8-0
+.. rst-class:: v3-0-0 v2-9-0-nRF54H20-1 v2-9-1 v2-9-0 v2-8-0
```

Comment:
as per earlier comment this needs to be removed
```suggestion
.. rst-class:: v2-9-0-nRF54H20-1 v2-9-1 v2-9-0 v2-8-0
```

Reply:
Whoops, overlooked that. Fixed now.